splunk-otel-android/src/main/java/io/opentelemetry/rum/internal/instrumentation/activity/Pre29ActivityCallbacks.java

```diff
@@ -14,18 +14,18 @@
  * limitations under the License.
  */
 
-package com.splunk.rum;
+package io.opentelemetry.rum.internal.instrumentation.activity;
 
 import android.app.Activity;
 import android.os.Bundle;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import io.opentelemetry.rum.internal.DefaultingActivityLifecycleCallbacks;
 
-class Pre29ActivityCallbacks implements DefaultingActivityLifecycleCallbacks {
+public class Pre29ActivityCallbacks implements DefaultingActivityLifecycleCallbacks {
```

Comment:
WDYT about moving the SDK-version check logic in to the OTel package; hiding it behind a factory method, or an `installOn` method (kinda like in other instrumentations)
Perhaps we could keep some of these classes package-private then

Reply:
Probably fine, can we do in a following PR?

Reply:
I have another branch that factors out the lifecycle instrumentation into the otel package in a way that's more consistent with the other `.installOn(app)` stuff. I think this is addressed there.